recipes/json-c/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: json-c
+    version: 0.12
+
+source:
+    fn: json-c-0.12.tar.gz
+    url: https://s3.amazonaws.com/json-c_releases/releases/json-c-0.12.tar.gz
+    md5: 3ca4bbb881dfc4017e8021b5e0a8c491
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - autoconf
+        - automake
+        - libtool
+    run:
+        - libgcc
```

Comment:
Thanks for the note. In the future, we will definitely be distributing a package like this in probably all cases where `gcc` is used. There are definitely a few cases where we have to now. GNU's OpenMP is definitely one. In this case, we are lucky not to need such things, but it is always good to keep an eye out. Thanks for the ref.